packages/vscode-extension/src/utilities/license.ts

```diff
@@ -0,0 +1,68 @@
+import fetch from "node-fetch";
+import { extensionContext } from "./extensionContext";
+import { exec } from "./subprocess";
+import { Logger } from "../Logger";
+import { simulatorServerBinary } from "./simulatorServerBinary";
+import { ActivateDeviceResult } from "../common/Project";
+
+const TOKEN_KEY = "token_key";
```

Comment:
where is this value stored? Maybe we can use some key that includes radon-ide in the name, such that it could be easier to locate the source of this data or avoid other extensions reading it?